Account/FxAClient10.swift

```diff
@@ -341,6 +378,63 @@ open class FxAClient10 {
 
         return makeRequest(mutableURLRequest, responseHandler: FxADevice.fromJSON)
     }
+    
+    open func oauthAuthorize(withSessionToken sessionToken: NSData, keyPair: RSAKeyPair, certificate: String) -> Deferred<Maybe<FxAOauthResponse>> {
+        let audience = self.getAudience(forURL: self.oauthURL)
+        let assertion = JSONWebTokenUtils.createAssertionWithPrivateKeyToSign(with: keyPair.privateKey,
+                                                                              certificate: certificate,
+                                                                              audience: audience,
+                                                                              issuer: "127.0.0.1",
```

Comment:
I took this from https://github.com/mozilla-mobile/firefox-ios/blob/02467f8015e5936425dfc7355c290f94c56ea57a/Account/FxAState.swift#L325, but I will confirm what we should use as an issuer.